.gitpod.yml

```diff
@@ -1,5 +1,12 @@
 tasks:
-  - init: mvn install -P quick-build
+  - init: |
+      # Install and set Java 21
+      curl -s "https://get.sdkman.io" | bash
```

Comment:
I thought [SDKMAN!](https://sdkman.io/) is included in the default base iamge?: https://www.gitpod.io/docs/introduction/languages/java#the-default-base-image

Do we really need to add an additional step to install it?